pkg/virt-operator/resource/apply/resourcemerge/resourcemerge.go

```diff
@@ -0,0 +1,131 @@
+package resourcemerge
```

Comment:
Was this file copied over from somewhere too? If so, maybe we can point to the source in a comment at the top of the file?

Reply:
I've added file name and link to where it was copied from.